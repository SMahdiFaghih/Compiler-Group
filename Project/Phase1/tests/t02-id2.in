A_SHOUTING_ID
a______________b
q999999999999000000000
