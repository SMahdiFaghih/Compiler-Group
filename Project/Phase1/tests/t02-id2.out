T_ID A_SHOUTING_ID
T_ID a______________b
T_ID q999999999999000000000
